### PR TITLE
Lint GitHub integration

### PR DESCRIPTION
**Pull request by victorhuangwq**
## Description

Ran rubocop on github_integration.rb & github_integrations_controller.rb

## Motivation and Context

Part of linting effots on autolab codebase

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation, which is located at [Autolab Docs](https://github.com/autolab/docs)
- [ ] I have updated the documentation accordingly, included in this PR

## Other issues / help required

If unsure, feel free to submit first and we'll help you along.

### CODE REVIEW

File: app/controllers/github_integrations_controller.rb [hunk 1]
```diff
@@ -21,20 +22,20 @@ def get_repositories
...
 
-  private
+private
 
   def set_github_integration
-    gi = current_user.github_integration
-    if gi && gi.is_connected
-      @github_integration = current_user.github_integration
-    end
+    return if !current_user&.github_integration&.is_connected
```
Comment (victorhuangwq) [inline app/controllers/github_integrations_controller.rb:36]: let me know if this is similar to what you intended.

File: app/models/github_integration.rb [hunk 2]
```diff
@@ -6,27 +6,27 @@ class GithubIntegration < ApplicationRecord
...
+                           query: {
+                             sort: "pushed",
+                             per_page: 30,
+                           })
+      repos.map { |repo|
+        { repo_name: repo[:full_name],
+          clone_url: repo[:clone_url],
+          default_branch: repo[:default_branch] }
+      }
     rescue Exception => e
```
Comment (victorhuangwq) [inline app/models/github_integration.rb:26]: Do we need to catch all exception here? or just standarderror

File: app/controllers/github_integrations_controller.rb (hunk 1 above)
Comment (fanpu) [reply app/controllers/github_integrations_controller.rb:36]: Could the if and negation be changed to an unless?

**Review by fanpu — DISMISSED** [review]
There are a couple of places using if ! <cond>, I think changing it to unless <cond> will be more idiomatic if we are going to change it anyway (https://github.com/rubocop/ruby-style-guide#if-vs-unless). Other than that everything else seems good!

File: app/controllers/github_integrations_controller.rb (hunk 1 above)
Comment (victorhuangwq) [reply app/controllers/github_integrations_controller.rb]: yup that make sense

File: app/models/github_integration.rb (hunk 2 above)
Comment (fanpu) [reply app/models/github_integration.rb:26]: Good catch, just StandardError. We don't want to deal with stuff raised internally by Ruby

**Review by fanpu — APPROVED** [review]

**Pull request merged**